docs/data/material/customization/dark-mode/dark-mode.md

```diff
@@ -229,6 +229,78 @@ import Button from '@mui/material/Button';
 </Button>;
 ```
 
+### API
+
+`theme.applyStyles(mode, styles) => CSSObject`
+
+Apply styles for a specific mode.
+
+#### Arguments
+
+- `mode` (`'light' | 'dark'`) - The mode for which the styles should be applied.
+- `styles` (`CSSObject`) - An object which contains the styles to be applied for the specified mode.
+
+#### Returns
+
+A `CSSObject` which key is the mode selector and value is the styles object: `{ [modeSelector]: styles }`.
+
+By default, the `modeSelector` is `'@media (prefers-color-scheme: light)'` for light mode and `'@media (prefers-color-scheme: dark)'` for dark mode.
+
+### Specificity
+
+Styles applied with `theme.applyStyles()` have higher specificity than the styles defined outside of it.
+This means that to override styles applied with it (via the `styled` function or `sx` prop), you need to use `theme.applyStyles()` as well:
+
+```jsx
+const BaseButton = styled('button')(({ theme }) =>
+  theme.applyStyles('dark', {
+    backgroundColor: 'white',
+  }),
+);
+
+const AliceblueButton = styled(BaseButton)({
+  backgroundColor: 'aliceblue', // In dark mode, backgroundColor will be white as theme.applyStyles() has higher specificity
+});
+
+const PinkButton = styled(BaseButton)(({ theme }) =>
+  theme.applyStyles('dark', {
+    backgroundColor: 'pink', // In dark mode, backgroundColor will be pink
+  }),
+);
+```
+
+### Overriding applyStyles
+
+If you need to override `theme.applyStyles()`, you can do so by providing a custom function.
```

Comment:
this feels strange given what comes right before it—we present a way to override this function in the Specificity section, then suggest a different way to do it here. Or I guess this is describing a more specific use case—overriding styles vs overriding the way the function itself works?

Reply:
Moved the last line which clarifies this to the top